### PR TITLE
[feat]tools-v2: add bs create volume snapshot

### PR DESCRIPTION
**Pull request by baytan0720**
### What problem does this PR solve?

Issue Number: #2585  

Problem Summary:

### What is changed and how it works?

What's Changed:
modify tools-v2/internal/utils/snapshot.go
modify tools-v2/internal/utils/row.go
modify tools-v2/pkg/cli/command/curvebs/create/create.go
modify tools-v2/pkg/config/bs.go
add tools-v2/pkg/cli/command/curvebs/create/volume/snapshot/snapshot.go
add tools-v2/pkg/cli/command/curvebs/create/volume/volume.go

How it Works:

Usage:
```bash
curve bs create volume snapshot --user root --filename test --snapshotname snap-test
```

Output:
```
+------+----------+--------------+
| USER | FILENAME | SNAPSHOTNAME |
+------+----------+---------------
| root |   test   |  snap-test   |
+------+----------+--------------+
```

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

File: tools-v2/pkg/cli/command/curvebs/create/volume/snapshot/snapshot.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	payload := map[string]any{}
+	if err := json.Unmarshal([]byte(result), &payload); err != nil {
+		return err
+	}
+	row := make(map[string]string)
+	row[cobrautil.ROW_USER] = sCmd.user
+	row[cobrautil.ROW_FILE_NAME] = sCmd.fileName
+	row[cobrautil.ROW_SNAPSHOTNAME] = sCmd.snapshotName
+
+	if payload["Code"] != "0" {
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/create/volume/snapshot/snapshot.go:112]: defined as constant

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Review by Cyber-SiKu — APPROVED** [review]

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by caoxianfei1** [comment]
@baytan0720 
image

**Comment by caoxianfei1** [comment]
@baytan0720 need to rebase.

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by wuhongsong** [comment]
@caoxianfei1 PTAL

File: tools-v2/README.md [hunk 2]
```diff
@@ -1852,6 +1853,24 @@ Output:
 +---------+
 ```
 
+##### create volume snapshot
+
+create snapshot
+
+Usage:
+```bash
+curve bs create volume snapshot --user root --filename test --snapshotname snap-test
```
Comment (caoxianfei1) [inline tools-v2/README.md:1862]: The filename is '/test'?

File: tools-v2/pkg/cli/command/curvebs/create/volume/snapshot/snapshot.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+		snapshotutil.QueryAction: snapshotutil.ActionCreateSnapshot,
+		snapshotutil.QueryUser:   sCmd.user,
+		snapshotutil.QueryFile:   sCmd.fileName,
+		snapshotutil.QueryName:   sCmd.snapshotName,
+	}
+	subUri := snapshotutil.NewQuerySubUri(params)
+	metric := basecmd.NewMetric(sCmd.snapshotAddrs, subUri, sCmd.timeout)
+	result, err := basecmd.QueryMetric(metric)
+	if err.TypeCode() != cmderror.CODE_SUCCESS {
+		return err.ToError()
```
Comment (caoxianfei1) [inline tools-v2/pkg/cli/command/curvebs/create/volume/snapshot/snapshot.go:103]: we should get error code and return proper error description.

**Review by caoxianfei1 — APPROVED** [review]
LGTM!

**Pull request merged**